pynest/nest/tests/test_refractory.py

```diff
@@ -134,14 +134,14 @@
 
 
 # --------------------------------------------------------------------------- #
-#  Simulation and refractory time
+#  Simulation and refractory time limits
```

Comment:
@Silmathoron, Sorry, I meant to change this to "Simulation time and refractory time limits"